binding/react-native/src/rhino_exceptions.tsx

```diff
@@ -0,0 +1,109 @@
+//
+// Copyright 2021 Picovoice Inc.
+//
+// You may not use this file except in compliance with the license. A copy of the license is located in the "LICENSE"
+// file accompanying this source.
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+// an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+// specific language governing permissions and limitations under the License.
+//
+
+class RhinoException extends Error {
```

Comment:
"Error" is the standard JS way of referring to exceptions right? We should probably call these JS errors `Rhino*Error` so that it doesn't look out of place